### PR TITLE
implemented join operation using inventory items

### PR DESCRIPTION
**Pull request by nuxeric**
I added in new items into the script so that the join operation on containerID 5 would be more visual

lmk if you have any questions thanks
image

### CODE REVIEW

**Comment by pseigo** [comment]
This looks nice :) Only thing is there's no way to get here from the main page

image

File: web/src/views/web/page/inventory/join.ejs [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+                    </tr>
+                <% }); %>
+            <% } %>
+            </tbody>
+        </table>
+
+        <p>This is the result of the following query:</p>
+        <pre>
+  SELECT I.serial_num, I.container_id, I.type_name, I.weight, I.manufacture_date
+  FROM inventory I, storage_container S WHERE S.id = I.container_id AND S.id = < Given Container ID >;</pre>
```
Comment (pseigo) [inline web/src/views/web/page/inventory/join.ejs:40]: maybe put on a new line :)

```suggestion
  FROM inventory I, storage_container S
  WHERE S.id = I.container_id AND
        S.id = <Given Container ID>;</pre>
```

You can make this change along with adding a button to /inventory at the site's index

**Review by pseigo — CHANGES_REQUESTED** [review]

File: web/src/views/web/page/inventory/join.ejs (hunk 1 above)
Comment (pseigo) [reply web/src/views/web/page/inventory/join.ejs:40]: Might also be nice to have a `style="button button-outline"` button on the query page to return to `/inventory`.

**Comment by ShaunDonn** [comment]
I have no suggested changes apart from what Peyton said. Looks good!

**Review by ShaunDonn — APPROVED** [review]

**Comment by nuxeric** [comment]
image

File: web/src/views/web/page/inventory/join.ejs (hunk 1 above)
Comment (nuxeric) [reply web/src/views/web/page/inventory/join.ejs]: Done, but through a separte commit as these changes are outdated

File: web/src/views/web/page/inventory/join.ejs [hunk 2]
```diff
@@ -36,8 +36,9 @@
 
         <p>This is the result of the following query:</p>
         <pre>
-  SELECT I.serial_num, I.container_id, I.type_name, I.weight, I.manufacture_date
-  FROM inventory I, storage_container S WHERE S.id = I.container_id AND S.id = < Given Container ID >;</pre>
+                SELECT I.serial_num, I.container_id, I.type_name, I.weight, I.manufacture_date
+                FROM inventory I, storage_container S
+                WHERE S.warehouse_id = < Given WarehouseID > AND I.container_id = S.id</pre>
```
Comment (pseigo) [inline web/src/views/web/page/inventory/join.ejs]: If you get rid of those extra spaces on the left it'll show properly on the page :)

Although if you put 0 spaces it looks like it's too close to the side lol so I've been putting 2 spaces. Tbh better thing to do is use 0 spaces and then fix the CSS but that's a problem for the future

**Review by pseigo — APPROVED** [review]

**Pull request merged**